qaframework-bdd-tests/src/test/resources/features/refapp-2.x/02-login/verifyLogin.feature

```diff
@@ -0,0 +1,10 @@
+Feature: User based login
+
+@selenium
+@defaultLogin
```

Comment:
how about renaming the tag  to `@login` ?

Reply:
Login is being used some where here  https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/RunTest.java#L27   so they will contradict. cucumber allows one single naming.